experiments/ClimaEarth/components/ocean/slab_ocean.jl

```diff
@@ -49,11 +49,11 @@ function slab_ocean_space_init(space, params)
     coords = CC.Fields.coordinate_field(space)
 
     # initial condition
-    T_sfc = CC.Fields.zeros(space) .+ params.T_init # FT(271) close to the average of T_1 in atmos
-    @. T_sfc += temp_anomaly(coords)
+    # FT(271) close to the average of T_1 in atmos
+    @. T_sfc = params.T_init + temp_anomaly(coords)
```

Comment:
this line fails because T_sfc isn't defined yet

Reply:
Ah, right, thanks!